NuCivic/dkan
# Fix dataset language display for POD validation CIVIC 6063

Issue: CIVIC-6063

## Description

1. POD validation fails on the language field, it is expecting a key value and is getting the label value.
2. The keys that exist are using underscores and the validator is expecting dashes. (i.e. en_US should be en-US)
3. The labels include commas that mess up the language array, converting those to dashes as well.

The value in the data.json doesn't conform to the standards here:
https://project-open-data.cio.gov/v1.1/schema/#language

## How to reproduce

- create a new dataset
- choose a language
- run the validator and you will see the errors

## QA Steps

- [x] edit a dataset
- [x] Enter a language or two into the *Language* field
- [x] POD check: run the validator and confirm there are no errors:  `admin/config/services/odsm/validate/pod`
- [x] Review 'data.json' output to confirm the languages are using keys
- [x] DCAT check: run the validator and confirm there are no errors: `admin/config/services/odsm/validate/dcat`
- [x] From the dataset page, click the 'RDF' link in the sidebar, confirm the languages are using keys

## Merge process

- [x] Merge https://github.com/NuCivic/open_data_schema_map/pull/87
- [x] Merge https://github.com/NuCivic/open_data_schema_map/pull/89
- [ ] ~~Update `drupal-org.make` to use the `7.x-1.x` branch of  open_data_schema_map~~

## Reminders
